doc/custom_nonconformity_score_proof.ipynb

```diff
@@ -0,0 +1,407 @@
+    "& \\leq \\sum_{j=1}^{n} \\mathbb{1}\\{ \\lvert R_{n+1,j} \\rvert > \\lvert R_{i}^{LOO} \\rvert \\}  = \\sum_{j=1}^{n} A_{n+1,j}\n",
+    "\\end{align}$$\n",
+    "\n",
+    "Or otherwise:\n",
+    "$$ Y_{n+1} < \\hat{q}_{n,\\alpha}^{+}\\{ g( - \\lvert R_{i}^{LOO} \\rvert , \\hat{\\mu}_{-i}(X_{n+1})) \\}$$\n",
+    "which implies that $Y_{n+1} < g( - \\lvert R_{i}^{LOO} \\rvert , \\hat{\\mu}_{-i}(X_{n+1}))$ for at least $(1-\\alpha)(n+1)$ many indices $j \\in \\{1,...,n\\}$.\n",
+    "$$\\begin{align}\n",
+    "(1-\\alpha)(n+1) & \\leq \\sum_{j=1}^{n} \\mathbb{1}\\{ Y_{n+1} < g( - \\lvert R_{i}^{LOO} \\rvert , \\hat{\\mu}_{-i}(X_{n+1})) \\}\\\\\n",
+    "& = \\sum_{j=1}^{n} \\mathbb{1}\\{ f( Y_{n+1} , \\hat{\\mu}_{-i}(X_{n+1}) ) < f( g( - \\lvert R_{i}^{LOO} \\rvert, \\hat{\\mu}_{-i}(X_{n+1})), \\hat{\\mu}_{-i}(X_{n+1})) \\} \\\\\n",
+    "& = \\sum_{j=1}^{n} \\mathbb{1}\\{ f( g(R_{n+1,j}, \\hat{\\mu}_{-j}(X_{n+1})) , \\hat{\\mu}_{-i}(X_{n+1}) ) < - \\lvert R_{i}^{LOO} \\rvert \\} \\\\\n",
+    "& = \\sum_{j=1}^{n} \\mathbb{1}\\{ R_{n+1,j} < - \\lvert R_{i}^{LOO} \\rvert \\} \\\\\n",
+    "& \\leq \\sum_{j=1}^{n} \\mathbb{1}\\{ \\lvert R_{n+1,j} \\rvert > \\lvert R_{i}^{LOO} \\rvert \\}  = \\sum_{j=1}^{n} A_{n+1,j}\n",
+    "\\end{align}$$\n",
+    "\n",
+    "Therefore, in either case, $n+1 \\in S(A)$, that is, point $n+1$ is a strange point with a high absolute conformity score.\n",
+    "\n",
+    "Thus:\n",
+    "$$\\mathbb{P}\\{ Y_{n+1} \\notin \\hat{C}_{n,\\alpha}^{jackknife+}(X_{n+1}) \\} \\leq 2\\alpha $$\n",
+    "and:\n",
+    "$$\\mathbb{P}\\{ Y_{n+1} \\in \\hat{C}_{n,\\alpha}^{jackknife+}(X_{n+1}) \\} \\geq 1-2\\alpha $$\n",
+    "\n",
+    "### Asymmetrical case\n",
+    "\n",
+    "Similarly to the theorem proof, let's suppose that $Y_{n+1} \\notin \\hat{C}_{n,\\alpha}^{jackknife+}(X_{n+1})$. This means either:\n",
+    "$$Y_{n+1}>\\hat{q}_{n,\\frac{\\alpha}{2}}^{+}\\{g(R_{i}^{LOO}, \\hat{\\mu}_{-i}(X_{n+1}))\\}$$\n",
+    "which implies that $Y_{n+1} > g(R_{i}^{LOO}, \\hat{\\mu}_{-i}(X_{n+1}))$ for at least $(1-\\frac{\\alpha}{2})(n+1)$ many indices $j \\in \\{1,...,n\\}$ since $g$ is an increasing function regarding the first variable,, or otherwise:\n",
+    "$$Y_{n+1}<\\hat{q}_{n,\\frac{\\alpha}{2}}^{-}\\{g(R_{i}^{LOO}, \\hat{\\mu}_{-i}(X_{n+1}))\\}$$\n",
+    "which implies that $Y_{n+1} < g(R_{i}^{LOO}, \\hat{\\mu}_{-i}(X_{n+1}))$ for at least $(1-\\frac{\\alpha}{2})(n+1)$ many indices $j \\in \\{1,...,n\\}$.\n",
+    "\n",
+    "Here $R_{j}^{LOO}$ is signed (it can be positive or negative) so it is added ($+$) in both cases. Besides, $\\frac{\\alpha}{2}$ is used instead of $\\alpha$ since to get the $(1-\\alpha)$ confidence interval, we look for an upper bound of the confidence interval so that $\\frac{\\alpha}{2}$ of the residuals are above and a lower bound of the confidence interval so that $\\frac{\\alpha}{2}$ of the residuals are below.\n",
```

Comment:
is signed (it can be positive or negative) -> can be positive or negative
$+$ -> useless
bound of -> bound on

Reply:
- Done
- Done
- Done